modules/nf-core/fastcov/main.nf

```diff
@@ -0,0 +1,46 @@
+process FASTCOV {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' ? 'docker://raverjay/fastcov:0.1.3--ba8c8cf6ae19': 'raverjay/fastcov:0.1.3--ba8c8cf6ae19' }"
```

Comment:
@fischer-hub We can build containers on top of the bioconda package: https://seqera.io/containers/